.travis.yml

```diff
@@ -40,3 +40,9 @@ notifications:
 branches:
   except:
     - /^*-v[0-9]/
+
+after_success:
+  - if [ "$TRAVIS_BRANCH" = "master" -a "$TRAVIS_PULL_REQUEST" = "false" ]; then
```

Comment:
I don't see using something like ![ -z $DOCKER_USER] && ![ -z $DOCKER_PASS]..
is that a miss?